config/webpack/environment.js

```diff
@@ -3,32 +3,28 @@
 
 // environment.config.set('resolve.extensions', ['.foo', '.bar']);
 // environment.config.set('output.filename', '[name].js');
-// environment.config.merge(customConfig);
-// environment.config.delete('output.chunkFilename');
+const { environment } = require('@rails/webpacker');
+const WebpackAssetsManifest = require('webpack-assets-manifest');
 
-// module.exports = environment;
+// Enable the default config
+environment.splitChunks(config =>
+  Object.assign({}, config, {
+    optimization: {
+      runtimeChunk: 'single',
+      splitChunks: {
+        cacheGroups: {
+          vendor: {
+            test: /[\\/]node_modules[\\/]/,
+            name: 'vendors',
+            chunks: 'all',
+          },
+        },
+      },
+    },
+  }),
+);
 
+// Remove the next lineif you want to transpile node modules
+environment.loaders.delete('nodeModules');
 
-const { environment } = require('@rails/webpacker')
-const webpack = require('webpack')
-
-environment.plugins.append(
-  'CommonsChunkVendor',
```

Comment:
These are deprecated in v4 in favour of the optimizations section. By default, webpack 4 generates a vendors bundle now, so no need for a configuration for this. See https://webpack.js.org/plugins/commons-chunk-plugin/